judge/forms.py

```diff
@@ -254,6 +256,30 @@ def clean(self):
             raise ValidationError(_('Must specify either totp_token or webauthn_response.'))
 
 
+class WCIPEGMergeActivationForm(Form):
```

Comment:
I would prefer this entire block of code to be at the end of the file. It'll be maintained as a patch on top of regular site code, and that would reduce the risk of conflicts.